workers/run_command.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+
+# Script for executing Django management commands within a Docker container.
+
+
+# This script should always run as if it were being called from
+# the directory it lives in.
+script_directory=`perl -e 'use File::Basename;
+ use Cwd "abs_path";
+ print dirname(abs_path(@ARGV[0]));' -- "$0"`
+cd $script_directory
+
+# However in order to give Docker access to all the code we have to
+# move up a level
+cd ..
+
+# Ensure that postgres is running
+if ! [[ $(docker ps --filter name=drdb -q) ]]; then
+    echo "You must start Postgres first with:" >&2
+    echo "./run_postgres.sh" >&2
+    exit 1
+fi
+
+volume_directory="$script_directory/volume"
+if [ ! -d "$volume_directory" ]; then
+    mkdir $volume_directory
+    chmod -R a+rwX $volume_directory
+fi
+
+source common.sh
+HOST_IP=$(get_ip_address)
+DB_HOST_IP=$(get_docker_db_ip_address)
+
+chmod -R a+rwX $volume_directory
+
+./prepare_image.sh -i smasher -s workers
+image_name=ccdlstaging/dr_smasher
```

Comment:
So the rest of this script seems like it could be used for most any image, but these lines make this smasher specific. It seems like these should be more generalized.